wmn-data.json

```diff
@@ -4004,6 +4028,18 @@
         "m_code" : 404,
         "known" : ["radio357", "radionowyswiat"],
         "cat" : "finance",
+        "valid" : true
+        },        
+        {
+        "name" : "Paypal",
+        "uri_check" : "https://www.paypal.com/paypalme/{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "userInfo",
+        "m_string" : "Get your very own PayPal.Me link",

```

Comment:
Let's try to make the `m_string` a component of the page not text shown in the browser as people that use languages other than English will get false positives.